.gitignore

```diff
@@ -162,3 +162,8 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# macOS Finder
+.DS_Store
+*.ignore.*
+*outputs*
```

Comment:
This feels like an overly broad inclusion into the root gitignore. 

If this is exclusion is important to your additions, consider adding a .gitignore with these lines closer to the code that creates these files

Reply:
yeah, makes sense.